src/components/deploymentConfig/service.tsx

```diff
@@ -61,6 +61,14 @@ export const updateDeploymentTemplate = (request, abortSignal) => {
     })
 }
 
+export const getIfLockedConfigProtected = (request) => {
+    return post(`${Routes.LOCKED_CONFIG_PROTECTED}`, request)
```

Comment:
we can directly write it as return post(Routes.LOCKED_CONFIG_PROTECTED, request)

Reply:
done